angel_system/data/load_bbn_medical_data.py

```diff
@@ -252,17 +252,18 @@ def save_as_kwcoco(classes, data, save_fn="bbn-data.mscoco.json"):
     dset.fpath = save_fn
     dset.dump(dset.fpath, newlines=True)
 
-    print_class_freq(dset)
+    # print_class_freq(dset)
```

Comment:
Should we just removed the commented out code?

Reply:
Going to merge this - I'll remove the comment in my own branch